# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1340741698 14400
#      Tue Jun 26 16:14:58 2012 -0400
# Branch stable
# Node ID f0552937a8eb9e17199b37c7b510ba2fcbef5db6
# Parent  b8041f48b0ff690968dcd515170f3eb20af2e359
doc: trivial doc fix

oop.txi: Remove stray \ in table.

diff --git a/doc/interpreter/oop.txi b/doc/interpreter/oop.txi
--- a/doc/interpreter/oop.txi
+++ b/doc/interpreter/oop.txi
@@ -544,17 +544,17 @@ width 0.6pt \tabskip=0pt\cr
 & $a .\hat b$ && power (a, b) && Element-wise power operator&\cr
 & $a \hat b$ && mpower (a, b) && Matrix power operator&\cr
 & $a < b$ && lt (a, b) && Less than operator&\cr
 & $a <= b$ && le (a, b) && Less than or equal to operator&\cr
 & $a > b$ && gt (a, b) && Greater than operator&\cr
 & $a >= b$ && ge (a, b) && Greater than or equal to operator&\cr
 & $a == b$ && eq (a, b) && Equal to operator&\cr
 & $a != b$ && ne (a, b) && Not equal to operator&\cr
-& $a \& b$ && and (a, b) && Logical and operator&\cr
+& $a & b$ && and (a, b) && Logical and operator&\cr
 & $a | b$ && or (a, b) && Logical or operator&\cr
 & $! b$ && not (a) && Logical not operator&\cr
 & $a'$ && ctranspose (a) && Complex conjugate transpose operator &\cr
 & $a.'$ && transpose (a) && Transpose operator &\cr
 & $a : b$ && colon (a, b) && Two element range operator &\cr
 & $a : b : c$ && colon (a, b, c) && Three element range operator &\cr
 & $[a, b]$ && horzcat (a, b) && Horizontal concatenation operator &\cr
 & $[a; b]$ && vertcat (a, b) && Vertical concatenation operator &\cr
